cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,73 @@
+import { faker } from '@faker-js/faker';
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const accountBalance = 5096;
+  const depositAmount = faker.number.int({ min: 500, max: 1000 });
```

Comment:
The method `faker.number.int` is incorrect. It should be `faker.number.integer` to generate a random integer.